src/dotnet/commands/dotnet-run/LocalizableStrings.cs

```diff
@@ -28,5 +32,23 @@ internal class LocalizableStrings
         public const string RunCommandExceptionMultipleProjects = "Specify which project file to use because {0} contains more than one project file.";
 
         public const string RunCommandAdditionalArgsHelpText = "Arguments passed to the application that is being run.";
+
+        public const string RunCommandExceptionCouldNotLocateALaunchSettingsFile = "The specified launch profile could not be located.";
+
+        public const string RunCommandExceptionCouldNotApplyLaunchSettings = "The launch profile \"{0}\" could not be applied.\n{1}";
+
+        public const string DefaultLaunchProfileDisplayName = "(Default)";
+
+        public const string UsingLaunchSettingsFromMessage = "Using launch settings from {0}...";
+
+        public const string LaunchProfileIsNotAJsonObject = "Launch profile is not a JSON object.";
+
+        public const string LaunchProfileHandlerCannotBeLocated = "The launch profile type '{0}' is not supported.";
+
+        public const string UsableLaunchProfileCannotBeLocated = "A usable launch profile could not be located.";
+
+        public const string UnexpectedExceptionProcessingLaunchSettings = "An unexpected exception occurred while processing launch settings:\n{0}";
```

Comment:
This message should be changed to reflect the *expected* situations where we're catching JsonException.